src/client/firefox/commands.js

```diff
@@ -177,7 +179,7 @@ function evaluate(script: Script, { frameId }: EvaluateParam) {
   });
 }
 
-function debuggeeCommand(script: Script) {
+function debuggeeCommand(script: Script): ?Promise<void> {
   tabTarget.activeConsole.evaluateJS(script, () => {}, {});
 
   if (!debuggerClient) {
```

Comment:
maybe this should be `Promise.resolve`

Reply:
On line 194, `Promise.resolve()` does not contain a parameter, so I'm giving a `void` type to `Promise<>`. Is this correct? 🤔